src/Be.Vlaanderen.Basisregisters.MessageHandling.AwsSqs.Simple/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,17 @@
+using System.Resources;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: ComVisible(false)]
+[assembly: AssemblyProduct("Vlaamse basisregisters")]
+[assembly: AssemblyDescription("Message handling with AWS SQS.")]
```

Comment:
@ArneD I copied the description from paket.template, but it was the same for all projects.